packages/angular/projects/ui-angular/package.json

```diff
@@ -18,8 +18,10 @@
     "aws-amplify": "^6.3.2"
   },
   "dependencies": {
+    "@angular/common": "^14.3.0",
     "@aws-amplify/ui": "6.4.1",
     "nanoid": "3.1.31",
+    "rxjs": "^7.8.1",
```

Comment:
Guessing this is why we are removing the devDeps for these two but are we sure that won't cause conflict with the consuming projects deps?

Reply:
Just verified that these packages aren't dependencies of angular 14. In that case it's probably safe to just go with the version we were using as dev deps right? 

`@angular/common` isn't a dependency of any of the packages we're using actually, so it does seem weird that it wasn't causing problems given it's imported in client code.